yarn-project/archiver/src/archiver/archiver_store_test_suite.ts

```diff
@@ -366,8 +369,9 @@ export function describeArchiverDataStore(testName: string, getStore: () => Arch
       });
 
       it('returns previously stored contract data', async () => {
-        await expect(store.getContractData(block.newContractData[0].contractAddress)).resolves.toEqual(
-          block.newContractData[0],
+        // Assuming the first (and only) contract data in the first TX
```

Comment:
Once again think it's better to have checks and errors here than these comments.